data/mods/innawood/professions.json

```diff
@@ -221,5 +221,147 @@
+          "gloves_leather",
+          "primitive_hammer",
+          "copper_knife",
+          "wicker_backpack",
+          "crucible_clay"
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "bra", "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "unemployed",
+    "name": "Survivor",
+    "description": "Some would say that there's nothing particularly notable about you, but you've survived, and that's more than most could say right now.",
+    "points": 0,
+    "items": {
+      "both": {
+        "items": [ "skirt_grass", "shirt_straw", "footrags", "straw_basket", "primitive_knife", "fire_drill" ],
+        "entries": [ { "item": "water_clean", "charges": 6, "container-item": "waterskin" } ]
+      }
+    }
+  },
+  {
+    "type": "profession",
+    "id": "tailor",
+    "name": "Tailor",
+    "description": "Tailoring may not seem like the most useful skill when the world has ended.  Most people wouldn't expect a simple tailor to live very long.  This is your opportunity to prove them wrong.",
+    "points": 3,
+    "skills": [ { "level": 3, "name": "tailor" }, { "level": 1, "name": "survival" } ],
+    "proficiencies": [
+      "prof_fibers",
+      "prof_fibers_rope",
+      "prof_leatherworking_basic",
+      "prof_leatherworking",
+      "prof_furriery",
+      "prof_closures",
+      "prof_closures_waterproofing",
+      "prof_cobbling",
+      "prof_millinery"
+    ],
+    "items": {
+      "both": {
+        "items": [
+          "straw_basket",
+          "straw_fedora",
+          "longshirt",
+          "pants",
+          "socks",
+          "mocassins",
+          "patchwork_scarf",
+          "awl_bone",
+          "bronze_shears"
+        ],
+        "entries": [ { "item": "needle_bone", "ammo-item": "plant_fibre", "charges": 200 } ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "bra", "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "lumberjack",
+    "name": "Lumberjack",
+    "description": "You're a lumberjack, and you're okay.  You felled trees before the world ended, and you suspect the undead aren't nearly as tough.",
+    "points": 0,
+    "skills": [ { "level": 2, "name": "swimming" } ],
+    "items": {
+      "both": {
+        "items": [ "pants", "socks", "mocassins", "straw_hat", "longshirt", "patchwork_scarf", "vest_leather", "rope_makeshift_6" ],
+        "entries": [ { "item": "copper_ax", "custom-flags": [ "auto_wield" ] } ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "bra", "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "musician",
+    "name": "Musician",
+    "description": "You nailed your solo, but the audience erupted into screams instead of applause.  You weren't able to grab much during the panic, but at least you have your flute in your basket.",
+    "points": 0,
+    "skills": [ { "level": 2, "name": "speech" } ],
+    "items": {
+      "both": {
+        "ammo": 100,
+        "magazine": 100,
```

Comment:
Pretty sure these are typos

Reply:
I just had it in there since the original musician profession did
```suggestion
```